test/extended/util/client.go

```diff
@@ -423,6 +423,23 @@ func (c *CLI) SetupWorkSpaceWithSpecificPath(serverURL string) {
 	e2e.Logf("Workspace %q has been fully provisioned.", c.currentWs.Name)
 }
 
+// ListWorkSpaces returns a list of WorkSpaces under the org workspace
+func (c *CLI) ListWorkSpaces(serverURL string) []string {
```

Comment:
```suggestion
func (c *CLI) ListWorkSpaces() []string {
```
Seems the input parameter in useless.